GDJS/Runtime/runtimeobject.ts

```diff
@@ -98,6 +102,7 @@ namespace gdjs {
       );
       this._averageForce = new gdjs.Force(0, 0, 0);
       this._behaviorsTable = new Hashtable();
+      this._effects = objectData.effects;
```

Comment:
You probably don't need and don't want to store effects from objectData.
These are the effects as they should be initially.

If you store this on all the objects, this means:
- all objects have a pointer to these, which consume memory.
- if I update the object data, thanks to the hot reload, all the objects are now having a pointer to the old effects data, which is a memory leak and outdated information.

This should actually not be needed. You only need this when you initialize the object, so just iterate on `objectData.effects`. The runtime effects are then stored in `_rendererEffects` (which contains the actual Pixi filters).